framework/src/media/audio/audio_manager.c

```diff
@@ -520,7 +497,7 @@ static audio_manager_result_t set_audio_volume(audio_card_type_t type, uint8_t v
 
 	fd = open(card->card_path, O_RDONLY);
 	if (fd < 0) {
-		meddbg("Failed to open device path : %s type : %d\n", card->card_path, type);
+		meddbg("\t\tFailed to open device path : %s type : %d\n", card->card_path, type);
```

Comment:
Why tab added here?

Reply:
It was a temporal modification. Reverted.